bin/ch/DbgController.js

```diff
@@ -781,6 +781,44 @@ var controllerObj = (function () {
                     'sources': sources
                 });
             },
+            dumpFunctionProperties: function (frameIdOrArrayOfIds, expandLevel) {
```

Comment:
>frameIdOrArrayOfIds, expandLevel [](start = 46, length = 32)

nit : you can use default to get rid of undefined checks below.

Reply:
Done